packages/core/src/amazonq/index.ts

```diff
@@ -26,6 +26,7 @@ export { init as testChatAppInit } from '../amazonqTest/app'
 export { init as docChatAppInit } from '../amazonqDoc/app'
 export { activateBadge } from './util/viewBadgeHandler'
 export { amazonQHelpUrl } from '../shared/constants'
+export * from './webview/ui/texts/constants'
```

Comment:
generally a good idea to give a name for these index.ts interfaces. that avoids conflicts, and also encourages useful "namespacing" in the consumer.
```suggestion
export * as webviewConstants from './webview/ui/texts/constants'
```